### PR TITLE
Add silent install option '--silent' & Misc fixes

### PR DESCRIPTION
**Pull request by g0tmi1k**
+ Add Silent/Quiet/Automated/Unattended Install option to `setup/bash.sh` (`--silent`).

+ Kali Linux doesn't have `wine64` in the repos, so trying to use it will causes errors. It now checks before calling.

### CODE REVIEW

File: setup/setup.sh [hunk 1]
```diff
@@ -174,59 +190,80 @@ func_python_deps(){
...
-    sudo chmod 755 -R /opt/pyinstaller-2.0/
+    sudo chmod 0755 -R /opt/pyinstaller-2.0/
   fi
 
   if [ $(uname -m) == 'x86_64' ]; then
 
     # Prepare Wine Directories
-    echo ' [*] Preparing Wine64 Directories'
+    echo ' [*] Preparing Wine64 Directories (#2)'
     mkdir -p ~/.wine64/drive_c/Python27/Lib/
```
Comment (ChrisTruncer) [inline setup/setup.sh]: Would this line also need to have the same modification you created on line 194? (mkdir -p ~/.wine/drive_c/Python27/Lib/site-packages/).  It looks like right below you're referencing the site-packages/ directory, but it wasn't created yet (but just wanted to verify).

Comment (g0tmi1k) [inline setup/setup.sh]: Agreed. Fixing.

**Comment by ChrisTruncer** [comment]
Awesome, thanks for sending this over!  Merged

**Comment by g0tmi1k** [comment]
:tada:

**Pull request merged**